napalm/base/base.py

```diff
@@ -1583,3 +1583,11 @@ def compliance_report(self, validation_file=None, validation_source=None):
         """
         return validate.compliance_report(self, validation_file=validation_file,
                                           validation_source=validation_source)
+
+    def canonical_int(self, interface):
```

Comment:
I think this probably should be a private method, for end-user calls people can just directly call what is in `helpers`.